react-example/src/Screens/Login/Login.less

```diff
@@ -47,9 +47,30 @@ form{
     color: #fff;
     border: none;
     border-radius: 5px;
-    padding: 15px;
+    height: 50px;
     font-size: 1.6rem;
     line-height: 1.9rem;
     cursor: pointer;
   }
+  button:hover{
+    box-shadow: 0 0 20px rgb(173, 170, 170);
+  }
+  .btn-load{
+    background-color: #05ac53;
+  }
+  .load{
```

Comment:
Just a heads up. We keep animations inside an `Animations` folder inside the `Less` folder. It's not documented and don't worry about that here, but just wanted to let you know that you'll find that in future projects.